src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,12 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition: 300ms;
```

Comment:
It's better to put transition in a variable because in header and in catalog the same values.

Reply:
ok